x/ibc/20-transfer/module.go

```diff
@@ -18,6 +18,7 @@ import (
 )
 
 const (
+	// ModuleName declares the name of the module
```

Comment:
Why is it ok that it's the SubModuleName?

Reply:
the `20-transfer` module is actually a sub module of `x/ibc`, but it does need to conform to the `AppModule` interface. I think this is copy pasta